app/templates/index/mygoals.hbs

```diff
@@ -1,5 +1,5 @@
-{{mygoals/selection-goals model=model type="working"}}
+{{component 'bepstore-goals@goal/goal-mine' model=model type="working"}}
```

Comment:
I agree with @bartist, mostly because the sorting and filtering of goals is done based on things that are out of scope in bepstore-ui-goals, such as the current user.